### PR TITLE
br: add backup file layout description

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/10622
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: br/backup-and-restore-design.md [hunk 1]
```diff
@@ -29,15 +29,50 @@ BR can generate the following types of backup files:
...
 - `cf` indicates the Column Family of RocksDB (`default` or `write` by default).
 
+When data is backed up to Amazon S3 or a network disk, the SST files are named in the format of `regionID_regionEpoch_keyHash_timestamp_cf`.
+
 ### Storage format of SST files
 
 - For details about the storage format of SST files, see [Rocksdb BlockBasedTable Format](https://github.com/facebook/rocksdb/wiki/Rocksdb-BlockBasedTable-Format).
 - For details about the encoding format of backup data in SST files, see [Mapping of table data to Key-Value](/tidb-computing.md#mapping-of-table-data-to-key-value).
+
+### Backup file layout
```
Comment (ran-huang) [inline br/backup-and-restore-design.md:48]:
```suggestion
### Backup file structure
```
I think [file structure](https://www.google.com/search?q=file+structure&newwindow=1&source=lnms&tbm=isch&sa=X&ved=2ahUKEwiBw9qCjfD5AhXjGTQIHfx9AfgQ_AUoAXoECAEQAw&biw=960&bih=944&dpr=2) is more accurate in this case.

File: br/backup-and-restore-design.md [hunk 2]
```diff
@@ -29,15 +29,50 @@ BR can generate the following types of backup files:
...
+.
+└── 20220621
+    ├── backupmeta
+    |—— backup.lock
+    ├── {storeID}-{regionID}-{regionEpoch}-{keyHash}-{timestamp}-{cf}.sst
+    ├── {storeID}-{regionID}-{regionEpoch}-{keyHash}-{timestamp}-{cf}.sst
+    └── {storeID}-{regionID}-{regionEpoch}-{keyHash}-{timestamp}-{cf}.sst
+```
+
+When you back up data to Amazon S3 or a network disk, SST files are stored in sub-directories based on the storeID. The layout is as follows:
```
Comment (ran-huang) [inline br/backup-and-restore-design.md:62]:
```suggestion
When you back up data to Amazon S3 or a network disk, the SST files are stored in sub-directories based on the storeID. The structure is as follows:
```

**Review by ran-huang — APPROVED** [review]

**Comment by MoCuishle28** [comment]
LGTM

**Pull request merged**